src/Components/Popular/index.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useEffect, useState } from "react";
+import "./styles.css";
+const axios = require("axios").default;
+
+function Popular() {
+  const [popular, setPopular] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await axios.get("/api/products");
+        if (res.status === 200) {
+          const products = res.data.products;
+          console.log(products);
```

Comment:
remove console log

Reply:
Thank you for pointing it out i will remove them